modules/create-crio-default-capabilities.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-crio-default-capabilities_{context}"]
+= Creating a drop-in file for CRI-O's default capabilities
+
+You can change some of the settings associated with the {product-title} CRI-O runtime for the nodes associated with a specific machine config pool (MCP). Using a `ContainerRuntimeConfigController` (CRCC) custom resource (CR), you set the configuration values and add a label to match the MCP. The MCO then rebuilds the `crio.conf` and `default.conf` configuration files on the associated nodes with the updated values.
```

Comment:
I think there is no CR with name `ContainerRuntimeConfigController`

Reply:
this sentence makes me a little confuse. maybe we don't have to mention the controller to the end user. they just need to know which CR can be used to change the config